tests/clickhouse-test

```diff
@@ -3762,6 +3762,8 @@ def main(args):
     else:
         print("All tests have finished.")
 
+    clickhouse_execute(args, "SYSTEM FLUSH LOGS trace_log")
```

Comment:
It should be created automatically when background logs are flushed. Need to figure out why it doesn't work automatically on MacOS.